lib/zendesk_api/resources.rb

```diff
@@ -650,6 +664,18 @@ class GroupMembership < Resource
 
     has User
     has Group
+
+    def self.cbp_path_regexes
+      [%r{groups/\d+/memberships$}]
+    end
+  end
+
+  class Group < Resource
+    has_many :memberships, :class => GroupMembership, :path => "memberships"
```

Comment:
Very minor, but why not? 😄 

```suggestion
    has_many :memberships, class: GroupMembership, path: "memberships"
```